### ISSUE TITLE
[Screen reader - DetailsList]: Screen reader is not providing the sorting information when focus is on interactive column headers for "Variants" table

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [OfficeDev/office-ui-fabric-react#11564]
User Experience:
Users who rely on screen reader will face difficulty if screen reader is not providing the sorting information when focus is on interactive column headers
 
Test Environment:
OS: Windows 10
Version: 1903(OS Build 19023.1)
Browser: Microsoft Edge (44.19023.1.0)
URL: https://fabricweb.z5.web.core.windows.net/oufr/7.76.2/#/examples/detailslist
Screen reader: Narrator
 
Repro Steps:
Step 1: Start Narrator
Step 2: Open the above URL in Edge browser.
Step 3: Navigate to "Variants" column header.
Step 4: Verify behavior of narrator when focus is on column headers 
 
Actual:
Screen reader is not providing the sorting information when focus is on interactive column headers for "Variants" table
 
Expected:
Screen reader should provide sorting information when focus is on interactive column headers for "Variants" table.
 
When focus is on "Name" column header, narrator should announce as "Name button column header column 2 of 5 consists sorting"

After activating "Name" header, narrator should announce as "Name button column header column 2 of 5 sorted in Ascending/Descending"

MAS Reference
MAS4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

![image](https://user-images.githubusercontent.com/53210862/71499900-9f35fc00-2888-11ea-9bb1-7070005f7538.png)

**Comment by micahgodbolt** [comment]
I can't find any examples of HTML tables/grids using columnheaders and aria-sort stating their sort direction upon focus. aria-sort is only stated when the value changes. 

Either this is the intended behavior (narrator intentionally leaves that information out) or there is a bug in Narrator. I don't believe there is any problem with our existing code.

If you can provide an example that displays the correct behavior I'm happy to look at it and come up with a fix, otherwise I feel this is working as intended.

**Comment by micahgodbolt** [comment]
In Material UI they append a hidden string to the header text to indicate direction

https://material-ui.com/components/tables/

![image](https://user-images.githubusercontent.com/1434956/71534375-4465ca00-28b3-11ea-954f-9fa577aa39ed.png)

**Comment by micahgodbolt** [comment]
Ok, a little more digging. It seems we've done the work to include a description with sorting information, but Narrator is not reading them in Edge. 

![image](https://user-images.githubusercontent.com/1434956/71535067-08813380-28b8-11ea-8e31-413305ed05cb.png)

Looking at a checkbox example using aria-describedby, it seems narrator is not reading those values as expected. 

http://www.oaa-accessibility.org/examplep/checkbox1/

I'll file this as a bug with Narrator team.
